packages/web/src/common/store/pages/audio-rewards/sagas.ts

```diff
@@ -526,7 +531,40 @@ function* fetchUserChallengesAsync() {
       }
     )
 
-    const userChallenges = challengesData.map(userChallengeFromSDK)
+    // Fetch monthly play counts from 2025
```

Comment:
could we put all this in an if-statement conditional on the play count milestone challenges existing?